FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -89,23 +98,104 @@ protocol SessionsProvider {
 
     super.init()
 
+    SessionsDependencies.dependencies.forEach { subscriberName in
```

Comment:
Where do these dependencies come from? May be I'm missing that part of the code in my eyes?

Reply:
This part isn't called yet - when we integrate with products, they will call this eg: https://github.com/firebase/firebase-ios-sdk/pull/10664/files#diff-3c7ed577af56820c03d0e93aaff77cc09d547a85837e6e57c66401ae85330033R195

Reply:
Perfect, that completes the loop for me. Like I mentioned, capturing this in a doc would very helpful.